### DISCUSSION TITLE
about  calculating  error

### DISCUSSION THREAD

**Discussion by Mangwuha** [Anura3D/Anura3D_OpenSource#118] [Q&A]
Dear developers and friends
I am a novice who has just started learning this software. While following the tutorial to learn the example of column collapse, I encountered the problem in the picture: fortl: severe (24) end of file during read, unit 100. During the process of learning the examples, I tried to ensure consistency with the operating steps in the tutorials and videos, including establishing models and setting parameters. I hope someone can help me solve this problem here. Thank you all!
![error24](https://github.com/Anura3D/Anura3D_OpenSource/assets/139300915/bcbce426-957d-4a82-9a97-41ba608fb747)

**Comment by murphy-john** [comment]
This usually means the code reached the end of a file while there was still information which needs to be read in. Double check you input all the correct parameters in the model. Have you tried to debug the code to see where the reading runs out of information?

Can you explain more about what you are trying to model? There have been some issues with models with multiple materials.

**Reply by Mangwuha** [reply]
Dear Sir
Thank you for your reply. I am studying the example of column collapse in the tutorial. I followed the steps in the tutorials and video demonstrations, including building a model, inputting parameters, and meshing. I'm sorry, I don't know how to program code. Could you please teach me how to check for code issues?
![a](https://github.com/Anura3D/Anura3D_OpenSource/assets/139300915/29ae7e58-8fe4-46ca-acb2-bdcd63c393c0)
![c](https://github.com/Anura3D/Anura3D_OpenSource/assets/139300915/4cb7ec4d-e414-486f-815c-c976afa67259)
![d](https://github.com/Anura3D/Anura3D_OpenSource/assets/139300915/075183d3-284b-42ad-afd6-180b80035169)
![e](https://g
![b](https://github.com/Anura3D/Anura3D_OpenSource/assets/139300915/d87faa78-17ea-4489-86f4-5dd45c370c5d)
ithub.com/Anura3D/Anura3D_OpenSource/assets/139300915/a7a6748c-23c4-4850-b2d6-92d0df18bb25)
![f](https://github.com/Anura3D/Anura3D_OpenSource/assets/139300915/660fd096-5ca8-494b-84b4-2f4854243cb9)

**Reply by FrancescaCeccato** [reply]
Did you check the element type? Somethimes the wron definition of normal/quadratic element gives this error.

**Reply by murphy-john** [reply]
You can find out more about that [here](https://github.com/Anura3D/Anura3D_OpenSource/discussions/29#discussion-4274028). That's a link to our page to help us help you by asking a complete question and it has a link on debugging.
